<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>manejarSitios</class>
 <widget class="QWidget" name="manejarSitios">
  <property name="windowModality">
   <enum>Qt::WindowModality::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>718</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>849</width>
    <height>718</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sitios</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../V_Escritorio/recursos/iconos/ic_Sitios.png</normaloff>../../V_Escritorio/recursos/iconos/ic_Sitios.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>861</width>
     <height>721</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(30, 30, 30);
color: rgb(255, 255, 255);
border:none;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>19</y>
      <width>611</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>ADMINISTRACIÓN DE SITIOS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>271</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="verticalSpacing">
      <number>13</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="ciudadLabel">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ciudad</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="ciudadComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
border-bottom: 2px solid rgb(0, 170, 255);
border-left:none;
border-right:none;
border-top:none;
background-color: rgb(45, 45, 45);
padding-left:5px;
padding-right:5px;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="sitioLabel">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Sitio</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="sitioComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
border-bottom: 2px solid rgb(170, 0, 255);
border-left:none;
border-right:none;
border-top:none;
background-color: rgb(45, 45, 45);
padding-left:5px;
padding-right:5px;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="agregarSitio">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>100</y>
      <width>101</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 0, 255);
color: rgb(0, 0, 0);
border-radius:5px;</string>
    </property>
    <property name="text">
     <string>  Añadir</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../recursos/iconos/ic_agregar.png</normaloff>../recursos/iconos/ic_agregar.png</iconset>
    </property>
   </widget>
   <widget class="QFrame" name="frame_sitio">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>791</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(30, 30, 30);
color: rgb(255, 255, 255);
border:none;
border-radius:5px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QPushButton" name="editarSitio">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>20</y>
       <width>97</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(0, 170, 255);</string>
     </property>
     <property name="text">
      <string> Editar</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../recursos/iconos/ic_editar.png</normaloff>../recursos/iconos/ic_editar.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="eliminarSitio">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>20</y>
       <width>97</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(0, 170, 255);</string>
     </property>
     <property name="text">
      <string> Eliminar</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../recursos/iconos/ic_eliminar.png</normaloff>../recursos/iconos/ic_eliminar.png</iconset>
     </property>
    </widget>
    <widget class="QLabel" name="img_Sitio">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>241</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(53, 53, 53);
border: 5px solid qconicalgradient(cx:0, cy:0, angle:135, stop:0 rgba(255, 255, 0, 69), stop:0.375 rgba(255, 255, 0, 69), stop:0.423533 rgba(251, 255, 0, 145), stop:0.45 rgba(247, 255, 0, 208), stop:0.477581 rgba(255, 244, 71, 130), stop:0.518717 rgba(255, 218, 71, 130), stop:0.55 rgba(255, 255, 0, 255), stop:0.57754 rgba(255, 203, 0, 130), stop:0.625 rgba(255, 255, 0, 69), stop:1 rgba(255, 255, 0, 69));
border-radius:0px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>300</y>
       <width>731</width>
       <height>211</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:10px;
background-color: rgb(0, 170, 255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <widget class="QPlainTextEdit" name="desc_Sitio">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>711</width>
        <height>191</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(54, 54, 54);color:white;border-radius:0px;</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="plainText">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QLineEdit" name="nombreSitio">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(53, 53, 53);
color: rgb(255, 255, 255);
border:0px</string>
     </property>
     <property name="text">
      <string>Nombre del sitio seleccionado</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>259</y>
       <width>711</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;</string>
     </property>
     <property name="text">
      <string>DESCRIPCIÓN</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="guardarCambios">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>220</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(0, 170, 255);</string>
     </property>
     <property name="text">
      <string>  Guardar cambios</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../recursos/iconos/ic_guardar.png</normaloff>../recursos/iconos/ic_guardar.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="seleccionarImagen">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>30</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(0, 170, 255);</string>
     </property>
     <property name="text">
      <string> Seleccionar imagen...</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../recursos/iconos/ic_imagen.png</normaloff>../recursos/iconos/ic_imagen.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
    <widget class="QWidget" name="formLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>140</y>
       <width>221</width>
       <height>56</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_Editar">
      <item>
       <widget class="QLabel" name="ciudadLabel_2">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Ciudad</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="ciudadComboBox_Editar">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>28</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
border-bottom: 2px solid rgb(0, 170, 255);
border-left:none;
border-right:none;
border-top:none;
background-color: rgb(45, 45, 45);
padding-left:5px;
padding-right:5px;
border-radius:0px;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="aviso_ciudadNoSeleccionada">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>90</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 0);</string>
    </property>
    <property name="text">
     <string>SELECCIONE UNA CIUDAD</string>
    </property>
   </widget>
   <widget class="QLabel" name="aviso_sitio">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>130</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 0);</string>
    </property>
    <property name="text">
     <string>SELECCIONE UN SITIO</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
